src/lib/accessibility/Behaviors/Menu/MenuItemBehavior.ts

```diff
@@ -7,10 +7,12 @@ const MenuItemBehavior: Accessibility = (props: any) => ({
     },
     anchor: {
       role: 'menuitem',
-      'aria-expanded': props['submenuOpened'],
       tabIndex: '0',
+      'aria-label': props['aria-label'],
+      'aria-labelledby': props['aria-labelledby'],
     },
   },
+  handledProps: ['aria-label', 'aria-labelledby', 'aria-expanded'],
```

Comment:
please remove aria-expanded if you are removing it from anchor part

Reply:
I missed this, wanted at first to add the same override as in tab and toolbar but I found that submenuOpened is probably a leftover as it's not being used. will remove this as well